client/stylesheets/_responsive.scss

```diff
@@ -603,6 +703,25 @@
     .homeCardImg {
         height: 110px !important;
     }
+
+    .searchResultsCardName p {
+        font-size: 0.2rem !important;
+        margin: 0 !important;
+    }
+
+    .searchResultsCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardImg` should be written in lowercase with hyphens